rerun_py/rerun_sdk/rerun/components/disconnected_space_ext.py

```diff
@@ -12,6 +12,22 @@
 class DisconnectedSpaceExt:
     """Extension for [DisconnectedSpace][rerun.components.DisconnectedSpace]."""
 
+    def __init__(
+        self: Any,
+        disconnected: bool = True,
```

Comment:
The field is called `is_disconnected` in the fbs so there's probably something wrong here.

EDIT: Yep, tests fail.